README.md

```
@@ -1,26 +1,152 @@
+   pnpm transpile && pnpm clean:git-updates && pnpm test:all-updates && pnpm dev
+   ```
+
+2. **Update After Code Changes**:
+
+   ```bash
+   # Transpile and regenerate all updates
+   pnpm transpile && pnpm test:all-updates
+   ```
+
+3. **Clean Start for Development**:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)